waffle/templatetags/waffle_tags.py

```diff
@@ -70,13 +70,13 @@ def flag(parser, token):
 
 @register.tag
 def switch(parser, token):
-    condition = lambda request, name: switch_is_active(name)
+    condition = lambda request, name: switch_is_active(name)  # noqa: E731
```

Comment:
is E731 no lambdas? we could swap these to be non-lambdas or allow lambdas in the setup.cfg config